tests/functional/u/used/used_before_assignment_typing.rc

```diff
@@ -0,0 +1,2 @@
+[testoptions]
+min_pyver=3.8
```

Comment:
This will skip a lot of tests on Py 3.7. An alternative would be to move the walrus operator test into `undefined_variable_py38.py`, which already tests for both `undefined-variable` and `used-before-assignment`, given that they're analyzed with extremely similar code paths.

Reply:
Thanks for pointing this out!